lgc/builder/DescBuilder.cpp

```diff
@@ -254,6 +254,26 @@ Value *DescBuilder::getStride(ResourceNodeType descType, unsigned descSet, unsig
   return getInt32(node->stride * sizeof(uint32_t));
 }
 
+// =====================================================================================================================
+// Returns the reloc string suffix for the given resource type.
+// This is used with `reloc::DescriptorOffset` and must match the parsing logic in RelocHandler.cpp.
```

Comment:
Add a blank line between parameter comments and function comments

// Function comments
//
// @param